### ISSUE TITLE
SSL handshake failure during preprovisioning

### ISSUE THREAD

**Issue by rmccue** [Chassis/Chassis#365]
```
==> default: Connecting to apt.puppetlabs.com (apt.puppetlabs.com)|52.84.207.195|:443... 
==> default: connected.
==> default: OpenSSL: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure
==> default: Unable to establish SSL connection.
==> default: dpkg: error processing puppetlabs-release-precise.deb (--install):
==> default:  cannot access archive: No such file or directory
==> default: Errors were encountered while processing:
==> default:  puppetlabs-release-precise.deb
```

Need to investigate this, but potentially apt.puppetlabs.com has shut off SSL v3 support and 12.04 is too out of date to support TLS v1.

This manifests itself eventually as:
```
==> default: Running Puppet with development.pp...
==> default: stdin: is not a tty
==> default: Could not parse options: invalid option: --hiera_config
```

**Comment by joshycube** [comment]
Checked it with Trusty and it works! 12.04 is too old.

**Comment by joshycube** [comment]
It works with Trusty. Precise must be too old.

**Comment by katmoody** [comment]
^^^ Didn't work with Trusty when I tried it but maybe I didn't change everything over I need to. :(

**Comment by rmccue** [comment]
This will almost certainly be fixed with #356, but in the meantime, we should ensure it always works. We can likely just switch to the http URL instead.
